### PR TITLE
Vendor in latest upstream containers/image

### PR DESCRIPTION
**Pull request by umohnani8**
Libpod is now stable with upstream containers/image
Fixes https://github.com/projectatomic/libpod/issues/28

Signed-off-by: umohnani8 <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@rhatdan @mheon PTAL

File: libpod/images/image_data.go [hunk 1]
```diff
@@ -88,7 +88,6 @@ func GetData(store storage.Store, name string) (*Data, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "error reading image %q", img.ID)
 	}
-	defer imgRef.Close()
```
Comment (mheon) [inline libpod/images/image_data.go]: Did something happen in c/image so that we don't need to close these anymore?

**Comment by mheon** [comment]
One question, otherwise LGTM pending tests

File: libpod/images/image_data.go (hunk 1 above)
Comment (umohnani8) [reply libpod/images/image_data.go]: yeah c/image has changed quite a bit, and the Close method for types.Image no longer exists.

**Comment by umohnani8** [comment]
Tests are green!

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by rhatdan** [comment]
Homu is hung at the moment, merging manually.

**Pull request merged**